docs/rules.md

```diff
@@ -1,9 +1,8 @@
 
 # Linter Rules
 
-This document is automatically published on [github pages](http://mozilla.github.io/addons-validator/).
```

Comment:
Please change this link to http://mozilla.github.io/addons-linter/ instead of removing it. We changed the name of the repo some time ago but looks like we forgot to update this URL!